rimage/transform/parallel_projection_test.go

```diff
@@ -0,0 +1,111 @@
+package transform
+
+import (
+	"os"
+	"testing"
+
+	"github.com/golang/geo/r3"
+	"go.viam.com/test"
+	"go.viam.com/utils/artifact"
+
+	pc "go.viam.com/rdk/pointcloud"
+	"go.viam.com/rdk/spatialmath"
+)
+
+// Helper function that makes and returns a PointCloud from an artifact path.
```

Comment:
I would clarify that the pointcloud is shortened to only include `numPoints` points.